sirepo/job_api.py

```diff
@@ -310,6 +310,22 @@ def _validate_and_add_sbatch_fields(self, request_content, compute_model):
         return request_content
 
 
+def begin_session():
+    try:
+        u = sirepo.auth.logged_in_user()
+    except sirepo.util.Reply:
+        # If we get any Sirepo generated errors (ex. user dir not found) then just ignore and don't
+        # try to start session (ex. agent would be started in user dir which may not exist).
+        return
+    API().request(
+        api_name='api_beginSession',
+        _request_content=PKDict(
+            uid=u,
+            userDir=str(sirepo.simulation_db.user_path(u)),
```

Comment:
Why not put this in the try/except as well?

Reply:
40fdc6ae4